src/commands/utility/search-my-points.js

```diff
@@ -55,40 +82,24 @@ module.exports = {
       const month =
         interaction.options.getInteger('month') || currentDate.getMonth() + 1;
       const user = interaction.options.getUser('user') || interaction.user;
-
       const channelsInput = interaction.options.getString('channels');
       const channels = channelsInput
         ? channelsInput.split(',').map((channel) => channel.trim())
         : [];
-
-      const targetStartDate = new Date(year, month - 1, 0);
-      const targetEndDate = new Date(year, month, 1);
-
-      const startDateStr = `${targetStartDate.getFullYear()}-${String(
-        targetStartDate.getMonth() + 1
-      ).padStart(2, '0')}-${String(targetStartDate.getDate())}`;
-      const endDateStr = `${targetEndDate.getFullYear()}-${String(
-        targetEndDate.getMonth() + 1
-      ).padStart(2, '0')}-01`;
-
-      const escapedUserId = `<@${user.id}>`;
-
       const channelQueryParts = channels.length
-        ? channels.map((channel) => `in:${channel}`).join(' ')
+        ? channels.map((ch) => `in:${ch}`).join(' ')
```

Comment:
Commonly shortening words make it look confusing

```suggestion
        ? channels.map((channel) => `in:${channel}`).join(' ')
```